agency_client/src/agency_client.rs

```diff
@@ -5,7 +5,7 @@ use crate::configuration::AgencyClientConfig;
 use crate::error::AgencyClientResult;
 use crate::{validation, AgencyClientError, AgencyClientErrorKind};
 
-#[derive(Clone)]
+#[derive(Clone, Serialize, Deserialize)]
```

Comment:
- This adds alternative approach how to serialize and deserialize `AgencyClient`. Current approach is to create the cloud agent, receive `AgencyClientConfig` and use that to build AgencyClient. Currently we need to care about compatibility changes made to of `AgencyClientConfig`, but if we add this alternative approach, suddenly we should also mind compatibility changes in AgencyClient itself.
- Additionally `AgencyClient` should not be serialized with `wallet_handle`, as that is runtime information. Well, that can be easily solved with serde's `skip`, but then my first argument is still intact.

Reply:
This is a crude workaround, my bad making a PR out of it.